packages/block-editor/src/hooks/background.js

```diff
@@ -147,10 +147,12 @@ export function BackgroundImagePanel( {
 		return null;
 	}
 
+	/*
 	const defaultControls = getBlockSupport( name, [
 		BACKGROUND_SUPPORT_KEY,
 		'__experimentalDefaultControls',
 	] );
+*/
```

Comment:
Tiny nit: looks like this can be removed now?

Reply:
Good spotting